pom.xml

```diff
@@ -74,7 +75,6 @@
 
         <maven.surefire.plugin.version>3.0.0-M7</maven.surefire.plugin.version>
         <maven.checkstyle.plugin.version>3.2.0</maven.checkstyle.plugin.version>
-        <maven.sonar.plugin.version>3.6.0.1398</maven.sonar.plugin.version>
```

Comment:
I would rather use certain version and update the version manually if needed than rely on the latest version. WDYT?

Reply:
I removed the reference to a specific version because we are using the cloud version of Sonar - which means IMO that we should use the latest version of the plugin to always have compatibility.
Of course, we can go back to declaring the version if you want.